my-app/src/components/OpenedPost/OpenedPost.tsx

```diff
@@ -0,0 +1,53 @@
+import React, {useState, useEffect} from 'react'
+import PostPage from './PostPage'
+
+interface IPost {
+  id: number,
+  image: string,
+  text: string,
+  date: string,
+  title: string,
+}
+
+const OpenedPost = () => {
+  const [posts, setPosts] = useState([]);
+
+  // const URL = 'https://studapi.teachmeskills.by/blog/posts/?limit=10'
+  const url = new URL('https://64e17765ab0037358818387e.mockapi.io/posts/posts')
+    url.searchParams.append("completed", 'false');
```

Comment:
I think you can avoid this by doing:

`https://64e17765ab0037358818387e.mockapi.io/posts/posts?limit=12&page=2&completed=true`